app/views/contact-method.html

```diff
@@ -24,7 +24,10 @@
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
       <form method="POST" action="check-answers">
-          <h1 class="govuk-heading-xl">Email Address</h1>
+          <h1 class="govuk-label-wrapper" id="emailQuestion"><label class="govuk-label govuk-label--xl" for="emailQuestion">
```

Comment:
Is it OK to put the opening `label` tag on the next line? 

Just helps readabilty for when we move everything over tho a Rails app 👍

Reply:
Also, does `emailQuestion` refer to anything in the rendered page?  I could be wrong, but I thought `for` had to refer to an `input` field in a label?

Reply:
634695f should resolve this

Reply:
I think a for can link to anything that has an `id` I think in the past IO've used them for aria descriptions on images or movies but it was totally redundant having it reference the direct parent.